types/constants.go

```diff
@@ -13,42 +13,70 @@ import (
 )
 
 var (
-	BlockFrequency         BlockHeight
-	BlockSizeLimit         = uint64(2e6)
+	// BlockFrequency {help}
+	BlockFrequency BlockHeight
+	// BlockSizeLimit {help}
+	BlockSizeLimit = uint64(2e6)
+	// ExtremeFutureThreshold {help}
 	ExtremeFutureThreshold Timestamp
-	FutureThreshold        Timestamp
-	GenesisBlock           Block
+	// FutureThreshold {help}
+	FutureThreshold Timestamp
+	// GenesisBlock is the first block of the block chain
+	GenesisBlock Block
 
-	// The GenesisID is used in many places. Calculating it once saves lots of
+	// GenesisID is used in many places. Calculating it once saves lots of
 	// redundant computation.
 	GenesisID BlockID
 
+	// GenesisSiafundAllocation {help}
 	GenesisSiafundAllocation []SiafundOutput
-	GenesisTimestamp         Timestamp
-	InitialCoinbase          = uint64(300e3)
-	MaturityDelay            BlockHeight
-	MaxAdjustmentDown        *big.Rat
-	MaxAdjustmentUp          *big.Rat
-	MedianTimestampWindow    = uint64(11)
-	MinimumCoinbase          uint64
+	// GenesisTimestamp is the timestamp when genesis block was mined
+	GenesisTimestamp Timestamp
+	// InitialCoinbase {help}
+	InitialCoinbase = uint64(300e3)
+	// MaturityDelay {help}
+	MaturityDelay BlockHeight
+	// MaxAdjustmentDown {help}
+	MaxAdjustmentDown *big.Rat
+	// MaxAdjustmentUp {help}
+	MaxAdjustmentUp *big.Rat
+	// MedianTimestampWindow {help}
+	MedianTimestampWindow = uint64(11)
+	// MinimumCoinbase {help}
+	MinimumCoinbase uint64
```

Comment:
```
MinimumCoinbase is the minimum coinbase reward for a block.
The coinbase decreases in each block after the Genesis block,
but it will not decrease past MinimumCoinbase.
```